install/kubernetes/helm/istio/templates/_podDisruptionBudget.tpl

```diff
@@ -1,8 +1,4 @@
 {{- define "podDisruptionBudget.spec" }}
-{{- if .minAvailable }}
-  minAvailable: {{ .minAvailable }}
+  minAvailable: 1
```

Comment:
It is only possible to set minAvailable or maxAvailable in the PDB, not both.

There is another PR that works to fix this in the queue.

Cheers
-steve